test-cases/cdc/cdc-replication-longevity.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 600
+
+user_prefix: 'cdc-replication-longevity'
+db_type: mixed_scylla
+
+use_legacy_cluster_init: false
+
+n_db_nodes: 3
+instance_type_db: 'i3.large'
+
+n_test_oracle_db_nodes: 1
+instance_type_db_oracle: 'i3.large'
+
+n_loaders: 1
+instance_type_loader: 'c4.large'
```

Comment:
please change to c5 as c4 getting deprecated and won't be available in all regions.